pkger/http_server_packages_deprecated.go

```diff
@@ -859,7 +860,7 @@ func newJSONEnc(w io.Writer) encoder {
 	return enc
 }
 
-func (s *HTTPServer) encResp(w http.ResponseWriter, r *http.Request, enc encoder, code int, res interface{}) {
+func (s *HTTPServerPackages) encResp(w http.ResponseWriter, r *http.Request, enc encoder, code int, res interface{}) {
 	w.WriteHeader(code)
 	if err := enc.Encode(res); err != nil {
 		s.api.Err(w, r, &influxdb.Error{
```

Comment:
is there an explanation of the deprecation? a comment at the top of this file giving a little bit of the context on deprecation would be super helpful to anyone who stumbles upon a file that references being deprecated

Reply:
there will be once I get the replacements in place